go.mod

```diff
@@ -315,3 +314,5 @@ require (
 	sigs.k8s.io/kustomize/kyaml v0.14.3-0.20230601165947-6ce0bf390ce3 // indirect
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.3 // indirect
 )
+
+require github.com/coreos/go-semver v0.3.1 // indirect
```

Comment:
i really hate `go mod tidy` doing this, new deps seems to be added to a new `require` block, not sure why

Reply:
I think it's because we have more than two `require` blocks.

Reply:
Fixed!